helm/cas-ciip-portal/values-wksv3k-test.yaml

```
@@ -4,14 +4,23 @@ route:
   insecure: true # temporary setting until cert is generated
 
 env:
-  smtpConnectionString: smtp://example.com
+  smtpConnectionString: smtp://cas-mailhog:1025
   senderEmail: no-reply-dev.cas@gov.bc.ca
-  adminEmail: matthieu@button.is
+  adminEmail: admin@ciip.test
+  appHost: https://ciip-test.pathfinder.gov.bc.ca # This should be different from the route for the production site (appHost will be https://ciip.gov.bc.ca)
 
 fider:
   host: https://ciip-portal-wksv3k-test-feedback.pathfinder.gov.bc.ca/
 # TODO move fider template in this repo
 
+mailhog:
+  enable: true
+  securityContext:
+    runAsUser: 1009960000
+    fsGroup: 1009960000
+    runAsNonRoot: true
+  host: https://cas-mailhog-wksv3k-test.pathfinder.gov.bc.ca/
```

The host shouldn't have a protocol
```suggestion
  host: cas-mailhog-wksv3k-test.pathfinder.gov.bc.ca
```